srcpkgs/python-icalendar/template

```diff
@@ -14,7 +14,7 @@ maintainer="Oliver Kiddle <okiddle@yahoo.co.uk>"
 license="BSD-2-Clause"
 homepage="https://github.com/collective/icalendar"
 distfiles="${PYPI_SITE}/i/icalendar/icalendar-${version}.tar.gz"
-checksum=07c2447a1d44cbb27c90b8c6a5c98e890cc1853c6223e2a52195cddec26c6356
+checksum=7e6fe7232622abe32d8f54d0936ffcd5a9087198a4c2f1ec1803a7dd9fdd979f
 alternatives="icalendar:icalendar:/usr/bin/icalendar2"
 
 do_check() {
```

Comment:
this is obsolete as it's [the default](https://github.com/void-linux/void-packages/blob/e36c2cd3e576f7ebe396829f103cd6510738bcfc/common/build-style/python3-module.sh#L26)